ufm-test-services/docker-compose.yml

```
@@ -0,0 +1,50 @@
+version: "3"
+
+services:
+  pushgateway:
+    image: prom/pushgateway
+    ports:
+      - "9091:9091"
+    restart: unless-stopped
+    read_only: true
+    security_opt:
+      - "no-new-privileges:true"
+
+  prometheus:
+    image: prom/prometheus:latest
+    container_name: prometheus
+    ports:
+      - "9090:9090"
+    volumes:
+      - ./prometheus.yml:/etc/prometheus/prometheus.yml
+    depends_on:
+      - pushgateway
+    command:
+      - '--config.file=/etc/prometheus/prometheus.yml'
+    read_only: true
+    security_opt:
+      - "no-new-privileges:true"
+
+  grafana:
+    image: grafana/grafana:latest
+    container_name: grafana
+    ports:
+      - "3000:3000"
+    depends_on:
+      - prometheus
+    environment:
+      # TODO - Replace with ENV variables to allow for better security
+      - GF_SECURITY_ADMIN_PASSWORD=adminpassword
```

```suggestion
      # TODO - Replace with ENV variables to allow for better security
      - GF_SECURITY_ADMIN_PASSWORD=${GF_SECURITY_ADMIN_PASSWORD:-adminpassword}
 ```
Can do this immediately